### ISSUE TITLE
Link hover color contrast is insufficient when block bg is dark

### ISSUE THREAD

**Issue by chalin** [google/docsy#1524]
Note that issues reported here **existed before the migration to BSv5**. Here is an example of the hover color for a link on the Docsy docs homepage:

<img width="342" alt="image" src="https://github.com/google/docsy/assets/4140793/50c562e1-3a21-4231-a069-c40cc847f542">

Related, but distinct issue:

- #1483

**Comment by chalin** [comment]
Oops, it seems that I reported this already! 🤷🏼‍♂️ Closing since it is a duplicate of #1482.
